### DISCUSSION TITLE
[photos] Upload only photos taken after activation date of backup

### DISCUSSION THREAD

**Discussion by dnred** [ente-io/ente#2614] [Enhancements]
Add an option to upload photos taken only after the activation date of the backup.

**Comment by RyannGalea** [comment]
Yes, this to me is a critical option. Just installed the app and my phone wants to back up over 3500 items which I already have backed up on NAS and GPhotos.

**Comment by cyrilf** [comment]
Yes, this is my main blocker for adopting this app too..
One option, could be to delete all the photos from my phone (the ones already backed up elsewhere) to make this work seamlessly, but that's not something I want to do for now.
So this option would be great!
